packages/web3-core-requestmanager/src/index.js

```diff
@@ -272,6 +275,29 @@ RequestManager.prototype.clearSubscriptions = function (keepIsSyncing) {
         this.provider.reset();
 };
 
+/**
+ * Evaluates WS close event
+ *
+ * @method _isCleanClose
+ *
+ * @returns {boolean}
+ */
+RequestManager.prototype._isCleanCloseEvent = function (event) {
+    return typeof event === 'object' &&
+           ([1000].includes(event.code) || event.wasClean === true);
+};
+
+/**
+ * Detects Ipc close error. The node.net module emits ('close', isException)
+ *
+ * @method _isIpcCloseError
+ *
+ * @returns {boolean}
+ */
+RequestManager.prototype._isIpcCloseError = function (event) {
```

Comment:
missing params in the funcDoc

Reply:
Is it possible to follow the JSDoc standard? (https://jsdoc.app/tags-param.html)